app/api/notes/[noteId]+api.ts

```diff
@@ -30,9 +30,10 @@ export async function GET(
       },
     );
   }
-  const note = await getNote(token, Number(noteId));
 
-  if (!note) {
+  const noteExists = await selectNoteExists(Number(noteId));
+
+  if (!noteExists) {
```

Comment:
since this is a 1-time usage variable, we can inline it in the `if` condition

Reply:
083f7f9c8d47c58fc2a88c08b78f66d7b041f7cb